setup.py

```diff
@@ -12,6 +12,9 @@
     print("warning: pypandoc module not found, could not convert Markdown to RST")
     long_description = ''
 
+with open('requirements.txt') as f:
+    required = f.read().splitlines()
```

Comment:
Would it be better to have multiple requirements files for different things? It just feels awkward to have packages like `coverage` and `mocks` as part of the requirements. Maybe maintain a separate `dev_requirements` file that people can look at when they are booting up?